dev/CommonStyles/TestUI/CommonStylesPage.xaml

```diff
@@ -233,6 +240,39 @@
                     </StackPanel>
                 </Grid>
             </StackPanel>
+
+            <TextBlock Text="CornerRadiusFilterConverter:" Margin="0, 20, 0, 0"/>
```

Comment:
I would just write an API test for this. API tests are simpler to write, easier to debug, more reliable and execute more quickly. So in cases where we are not testing user interaction we should default to API tests.